index.js

```diff
@@ -138,6 +138,7 @@ function bootstrap(options) {
 
   app.use(userMiddleware);
   app.use(hofMiddleware.cookies());
+  app.use(require('hof-middleware-markdown')());
```

Comment:
O yes I see that's in a function and it is short 👍 

I think when in doubt, I'll put it separately